### PR TITLE
setup project

### PR DESCRIPTION
**Pull request by Muneerah1612**
**List of tasks completed**

-   [x] Specify what tasks are completed in this PR
- setup project environment
-   [ ] Specify what's left to be done

**How can this be tested?**

- Highlight how this PR can be tested
run "pip install -r requirements.txt",
run "python manage.py runserver"
**Additional info**
ensure that your secret key is saved in your .env file
activate the virtual environment before running pip install
Add more info if any. eg fixes #.

### CODE REVIEW

File: analyzer/settings.py [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+        'ENGINE': 'django.db.backends.sqlite3',
+        'NAME': BASE_DIR / 'db.sqlite3',
+    }
+}
+
+
+# Password validation
+# https://docs.djangoproject.com/en/3.1/ref/settings/#auth-password-validators
+
+AUTH_PASSWORD_VALIDATORS = [
```
Comment (toshmanuel) [inline analyzer/settings.py:90]: this is not needed

File: analyzer/urls.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
+"""analyzer URL Configuration
```
Comment (toshmanuel) [inline analyzer/urls.py:1]: line 1 to 14 can be removed

**Review by toshmanuel — CHANGES_REQUESTED** [review]

File: analyzer/urls.py (hunk 2 above)
Comment (Muneerah1612) [reply analyzer/urls.py:1]: Noted

File: analyzer/settings.py (hunk 1 above)
Comment (Muneerah1612) [reply analyzer/settings.py:90]: Noted

**Review by toshmanuel — APPROVED** [review]

**Pull request merged**